base/strings/io.jl

```diff
@@ -377,17 +379,22 @@ julia> unescape_string("\\\\u03c0") # unicode
 
 julia> unescape_string("\\\\101") # octal
 "A"
+
+julia> unescape_string("aaa \\g \\n", ['g']) # using `keep` argument
+"aaa \\g \n"
```

Comment:
The CI failure is due to this. Not sure exactly why. Maybe it needs to be `\\n`. You can check this locally by running `make -C doc doctest=true`.

Reply:
``` unescape_string("aaa \\g \\n", ['g'])``` 
results in
```"aaa \\g \n"```
I think there is a problem with docstrings here. I am not sure on how to fix that.

Reply:
This is the error.
```
julia> unescape_string("aaa \n \g", ['g']) # using `keep` argument
"aaa 
 \g"

> Subexpression:

    unescape_string("aaa \n \g", ['g']) # using `keep` argument

> Output Diff:

"aaa 
 \g"
```

I don't know why ```unescape_string("aaa \\g \\n", ['g'])``` interpreted as ```unescape_string("aaa \g \n", ['g'])``` . I think there is some issue with docstrings. I am not sure how to fix this.

Reply:
Have you tried writing `\\\\g` and so on?

Reply:
Thanks for the suggestion @nalimilan . I added the change.